api/channel.go

```diff
@@ -209,7 +209,7 @@ func updateChannel(c *Context, w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
-		if !strings.Contains(channelMember.Roles, model.CHANNEL_ROLE_ADMIN) && !strings.Contains(teamMember.Roles, model.ROLE_TEAM_ADMIN) {
```

Comment:
There is a context function `IsTeamAdmin` that checks both for team and system admin. I think we don't use channel roles anymore so that part should be removed.